user_docs/en/userGuide.t2t

```diff
@@ -1220,12 +1220,12 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
-==== Report line indents With ====
-the Report line indents with combo box has four options.
+==== Report line indents with ====
+This option allows you to configure how indentation at the beginning of lines is reported. The Report line indents with combo box has four options.

```

Comment:
nit: Please split these two sentences onto separate lines. We do one sentence per line in our documentation because it makes life easier for translators.